Purpose: Fix format warnings of manual page
Fixes:   lintian - manpage-has-errors-from-man
Status:  Not yet submitted upstream.
---
Index: insserv/insserv.8.in
===================================================================
--- insserv.orig/insserv.8.in	2009-06-26 12:34:19.000000000 +0200
+++ insserv/insserv.8.in	2009-06-26 12:34:19.000000000 +0200
@@ -12,11 +12,9 @@
 @@BEGIN_SUSE@@
 .TH INSSERV 8 "Jul 29, 2008" "Version 1.11" "The SuSE boot concept"
 .UC 8
-.OS SuSE Linux
 @@ELSE_SUSE@@
 .TH INSSERV 8 "Jul 29, 2008" "Version 1.11"
 .UC 8
-.OS Debian
 @@END_SUSE@@
 .SH NAME
 insserv \- Enable an installed system init script
@@ -35,9 +33,9 @@
 .RB [ \-c\ <config> ]
 .RB [ \-p\ <path> ]
 @@BEGIN_SUSE@@
-.RI [[ / ] path/to/init.d/ ] script [ ,start=<lvl1> [ ,<lvl2>\ ... ]
+.RI [[ / ] path/to/init.d/ ] script [ ,start=<lvl1,lvl2,...> ]
 @@ELSE_SUSE@@
-.RI [[ / ] path/to/init.d/ ] script [ ,start=<lvl1> [ ,<lvl2>\ ... ] ,stop=<lvl1> [ ,<lvl2>\ ... ]]
+.RI [[ / ] path/to/init.d/ ] script [ ,start=<lvl1,lvl2,...> ,stop=<lvl1,lvl2,...> ]
 @@END_SUSE@@
 .PP
 .B insserv
@@ -129,7 +127,7 @@
 should be started
 .B before
 the specified service names.
-Wereas the optional
+Whereas the optional
 .B X\-Stop\-After
 keyword implies that the script using this keyword
 should be stopped
@@ -223,7 +221,7 @@
 a passphrase or password input during boot
 or runlevel change.
 .P
-Beside the defined
+In addition to the defined
 .B System Facilities
 in the configuration file
 .IR /etc/insserv.conf ,
@@ -246,8 +244,8 @@
 directory.  Useful for testing.
 .TP
 .BR \-o\ <path> ,\  \-\-override\ <path>
-Path to replace existing LSB comment headers with the comment
-headers found in this path (default path is
+LSB comment headers found in this path will override existing
+LSB comment headers of scripts in the init.d directory (default path is
 .IR /etc/insserv/overrides/ ).
 .TP
 .BR \-p\ <path> ,\  \-\-path\ <path>
@@ -260,7 +258,7 @@
 Remove the listed scripts from all runlevels.
 .TP
 .BR \-d ,\  \-\-default
-Use default runlevels a defined in the scripts.
+Use default runlevels as defined in the scripts.
 This may restore an edited runlevel link scheme.
 .TP
 .BR \-f ,\  \-\-force
@@ -303,17 +301,16 @@
 For the runlevels the information found in
 the script is used.
 .TP
-.RI [[ / ] path/to/init.d/ ] script [ ,start=<lvl1> [ ,<lvl2> ]]\ ...
+.RI [[ / ] path/to/init.d/ ] script [ ,start=<lvl1,lvl2,...> ]
 List of scripts which have to be added to
 the specified runlevels to be started with.
 You may use this extension to override the default values
 for start and stop runlevels of the script.
 Note that
-.BR <lvl1> ,\  <lvl2> ,\ ...
+.BR lvl1 ,\  lvl2 ,\ ...
 are the known runlevels explained above.
 The extension
-.IR ,stop=<lvl1> [ ,<lvl2> ]]
-is possible
+.IR ,stop=<lvl1,lvl2,...>
 @@BEGIN_SUSE@@
 is possible but ignored on SuSE Linux.
 @@ELSE_SUSE@@
@@ -330,9 +327,9 @@
 .\"
 .SH OVERRIDES
 Beside using the extensions
-.IR ,start=<lvl1> [ ,<lvl2> ]]
+.IR ,start=<lvl1,lvl2,...>
 and
-.IR ,stop=<lvl1> [ ,<lvl2> ]]
+.IR ,stop=<lvl1,lvl2,...>
 it is possible to use override files replace a LSB comment header
 or simple provide a missing LSB comment header.  This can be done
 by placing a file with the new LSB comment header using the same
@@ -385,7 +382,9 @@
 .sp 1
 will be ignored.
 .SH BUGS
-Boot script sometimes misses comments.
+Boot scripts sometimes lack a LSB comment header. Contact a package
+maintainer or developer of the software which provides the script to
+have a LSB comment header added to it.
 .SH FILES
 .TP
 .I /etc/insserv.conf
